project/graph_utils.py

```diff
@@ -0,0 +1,24 @@
+import cfpq_data
+from collections import namedtuple
+from typing import Tuple
+import networkx
```

Comment:
Старайтесь разделять друг от друга импорты `from` и `import`
```suggestion
from collections import namedtuple
from typing import Tuple

import cfpq_data
import networkx
```